src/algebra/group_power/basic.lean

```diff
@@ -41,103 +41,35 @@ universes u v w x y z u₁ u₂
 variables {M : Type u} {N : Type v} {G : Type w} {H : Type x} {A : Type y} {B : Type z}
```

Comment:
The module docstring for this file is now out of date; it no longer provides the instances it claims to.